package.json

```diff
@@ -68,11 +68,12 @@
 	},
 	"dependencies": {
 		"@babel/runtime": "^7.17.0",
+		"cookie-session": "^2.0.0",
 		"express": "^4.17.2",
 		"express-session": "^1.17.3",
```

Comment:
Can you remove `express-session` now you've swapped to `cookie-session`?

Reply:
Make sure the login flow still works and shows `authenticated: true` in the `isAuthenticated` API afterwards though!